samples/sample-bugsnag/settings.gradle.kts

```diff
@@ -11,3 +11,13 @@
 include(":app", ":shared")
 
 enableFeaturePreview("VERSION_CATALOGS")
+
+includeBuild("../..") {
+    dependencySubstitution {
```

Comment:
Is substitution block required? I got the impression from [these docs](https://docs.gradle.org/current/userguide/composite_builds.html#included_build_declaring_substitutions) that it's only needed if the group doesn't match for some reason

Reply:
i dont think it is, i was able to use it without the substituion block for kmmbridge